spel/minimal-linux.pkr.hcl

```diff
@@ -476,6 +476,18 @@ variable "amigen7_storage_layout" {
 # Variables used by AMIgen8
 ###
 
+variable "amigen8_extra_rpms" {
+  description = "List of package specs (rpm names or URLs to .rpm files) to install to the EL8 builders and images"
+  type        = list(string)
+  default = [
+    "python39",
```

Comment:
Ok, cool. This gets us ahead of the deprecation of 3.6 _and_ 3.8.

Reply:
Hopefully RedHat will provide packages for python310 also. It's a nice upgrade from 3.9.